### PR TITLE
chat ui first version

### PR DESCRIPTION
**Pull request by manojk242003**
Screenshot 2024-01-31 at 11 38 14 PM
Screenshot 2024-01-31 at 11 37 57 PM

### CODE REVIEW

**Comment by royalpinto007** [comment]
Can you please use the Prettier extension to format the file you have edited so that the CI is green? @manojk242003

**Review by royalpinto007 — CHANGES_REQUESTED** [review]

**Comment by royalpinto007** [comment]
Please resolve the conflicts!

**Comment by manojk242003** [comment]
Fixing the chat UI

**Comment by royalpinto007** [comment]
Thanks, but the CI is still not green, please fix that too!

File: client/screens/Chat.js [hunk 1]
```diff
@@ -1,98 +1,145 @@
...
+        showAvatarForEveryMessage={false}
+        showUserAvatar={false}
+        onSend={messages => onSend(messages)}
+        messagesContainerStyle={{
+          backgroundColor: '#EEE7E1', // Set the background color to transparent
+          borderRadius: 20,
+        }}
+        textInputStyle={{
+          backgroundColor: 'white',
+          borderRadius: 20, // Set the border radius directly
```
[the remark this answers is not in the document]
Comment (royalpinto007) [reply client/screens/Chat.js]: Closing- https://github.com/EmberTechWorks/Ember/pull/14/commits/5ee06955d3d33c3913ecf36c19edf6d15bcbe632

File: client/screens/Chat.js [hunk 2]
```diff
@@ -1,98 +1,145 @@
...
+
+  return (
+
+      <GiftedChat
+        messages={messages}
+        showAvatarForEveryMessage={false}
+        showUserAvatar={false}
+        onSend={messages => onSend(messages)}
+        messagesContainerStyle={{
+          backgroundColor: '#EEE7E1', // Set the background color to transparent
```
[the remark this answers is not in the document]
Comment (royalpinto007) [reply client/screens/Chat.js]: Closing- https://github.com/EmberTechWorks/Ember/pull/14/commits/5ee06955d3d33c3913ecf36c19edf6d15bcbe632

**Pull request merged**